tests/test_suite.bats

```diff
@@ -4,6 +4,8 @@ TMP_GENERATION_DIR="${BATS_TEST_DIRNAME}/tmp"
 
 export TMP_GENERATION_DIR="${BATS_TEST_DIRNAME}/tmp"
 
+[ -z "$DOCKER_IMAGE_NAME_TO_TEST" ] && export DOCKER_IMAGE_NAME_TO_TEST=asciidoctor/docker-asciidoctor
```

Comment:
Would it be possible to invert the syntax to be sure an error is not triggered?
Because if this shell code is used in a context with the option `set -e` enabled, then it would fail.

aka.

```
[ -n "${DOCKER_IMAGE_NAME_TO_TEST}" ] || export DOCKER_IMAGE_NAME_TO_TEST=asciidoctor/docker-asciidoctor
```